radian/rutils.py

```diff
@@ -78,6 +78,14 @@ def load_custom_key_bindings(*args):
     for m in esc_keymap:
         map_key(("escape", m["key"]), m["value"], mode=m["mode"] if "mode" in m else "r")
 
+    keymap = roption("radian.ctrl_key_map", [])
+    for m in keymap:
+        if m["key"] in "mihdc":
+            print("WARNING: Cannot remap c-" + m["key"] + ". Please remove this mapping from radian.ctrl_key_map in your .radian_profile")
```

Comment:
The setting could be specified in various places. Could you rename .radian_profile to just "radian profile".

Reply:
Done!